### PR TITLE
Merge autocomplete

### PR DESCRIPTION
**Pull request by mcmadbat**
- better buttons
- autocomplete 
- using sample (and incomplete) data

### CODE REVIEW

File: chrome/src/js/autocomplete.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      results: function() {}
+    },
+    focus: function( event, ui ) {
+      $( "#nameInput" ).val( ui.item.name )
+      return false
+    },
+    select: function( event, ui ) {
+      $( "#nameInput" ).val( ui.item.name )
+      $( "#playerId" ).val( ui.item.id )
+      // $( "#player-description" ).html( ui.item.desc )
```
Comment (sidhantmishra) [inline chrome/src/js/autocomplete.js]: Remove this my dude

File: chrome/src/js/autocomplete.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    '3B',
+    'SS',
+    'LF',
+    'CF',
+    'RF',
+    'DH'
+  ]
+
+  $( "#nameInput" ).autocomplete({
+    minLength: 2,
```
Comment (sidhantmishra) [inline chrome/src/js/autocomplete.js:19]: Is this enough to make sure we're not getting many dupes?

File: chrome/src/js/autocomplete.js (hunk 1 above)
Comment (mcmadbat) [reply chrome/src/js/autocomplete.js]: we might add it in later

File: chrome/src/js/autocomplete.js (hunk 2 above)
Comment (mcmadbat) [reply chrome/src/js/autocomplete.js:19]: thats not what this does. it is how many characters before the autocomplete kicks in

**Pull request merged**